app/controllers/about_controller.rb

```diff
@@ -20,7 +20,7 @@ def supporter_gobridge
 
   private
   def use_num_individual_supporters
-    @num_individual_supporters = User::AcquiredBadge.joins(:user).includes(:user).
+    @num_individual_supporters = User::AcquiredBadge.includes(:user).
       where(badge_id: Badge.find_by_slug!("supporter")). # rubocop:disable Rails/DynamicFindBy
```

Comment:
Out of curiosity, why don't we do:

```suggestion
      where(badge_id: Badge.find_by!(slug: "supporter")).
```

Reply:
Good question! `find_by_slug` is a specific method on `Badge`.

<img width="644" alt="Screenshot 2022-08-18 at 12 56 11" src="https://user-images.githubusercontent.com/286476/185388810-dc756cb5-9414-4a39-932c-6811270c0fd1.png">

Reply:
Ah, I was confused. Could we use `for_slug!` instead to reduce the ambiguity?